api/capi/include/nnstreamer.h

```diff
@@ -569,6 +574,67 @@ int ml_pipeline_valve_release_handle (ml_pipeline_valve_h valve_handle);
  */
 int ml_pipeline_valve_set_open (ml_pipeline_valve_h valve_handle, bool open);
 
+/********************************************************
+ ** NNStreamer Element Property Control in Pipeline    **
+ ********************************************************/
+
+/**
+ * @brief Gets an element handle in NNStreamer pipelines to control its properties.
+ * @since_tizen 6.0
+ * @remarks If the function succeeds, @a elm_h handle must be released using ml_pipeline_element_release_handle().
+ * @param[in] pipe The pipeline to be managed.
+ * @param[in] element_name The name of element to control.
+ * @param[out] elm_h The element handle.
+ * @return @c 0 on success. Otherwise a negative error value.
+ * @retval #ML_ERROR_NONE Successful
+ * @retval #ML_ERROR_NOT_SUPPORTED Not supported.
+ * @retval #ML_ERROR_INVALID_PARAMETER Given parameter is invalid.
+ * @retval #ML_ERROR_OUT_OF_MEMORY Failed to allocate required memory.
+ */
+int ml_pipeline_element_get_handle (ml_pipeline_h pipe, const char *element_name, ml_pipeline_element_h *elm_h);
+
+/**
+ * @brief Releases the given element handle.
+ * @since_tizen 6.0
+ * @param[in] elm_h The handle to be released.
+ * @return @c 0 on success. Otherwise a negative error value.
+ * @retval #ML_ERROR_NONE Successful
+ * @retval #ML_ERROR_NOT_SUPPORTED Not supported.
+ * @retval #ML_ERROR_INVALID_PARAMETER Given parameter is invalid.
+ */
+int ml_pipeline_element_release_handle (ml_pipeline_element_h elm_h);
+
+/**
+ * @brief Sets element properties in NNStreamer pipelines.
+ * @since_tizen 6.0
+ * @remarks This function supports a varying number of name/value pairs, finished by NULL.
+ * @remarks If one of given property name does not exist, all properties are not set.
+ * @param[in] elm_h The target element handle.
+ * @param[in] first_property_name The name of first property to set.
+ * @param[in] ... A varying number of name/value pairs, finished by NULL.
+ * @return @c 0 on success. Otherwise a negative error value.
+ * @retval #ML_ERROR_NONE Successful
+ * @retval #ML_ERROR_NOT_SUPPORTED Not supported.
+ * @retval #ML_ERROR_INVALID_PARAMETER Given parameter is invalid or given property name does not exist.
+ */
+int ml_pipeline_element_set_property (ml_pipeline_element_h elm_h, const char *first_property_name, ...);
```

Comment:
Need more description (examples) on how to use get/set properties in Doxygen entries.

Reply:
I added the code example for doxygen entry.

In my humble opinion, it is enough since this implementation supports both single property and multiple properties to control. Tomorrow I will talk with Jaeyun and you about this issue.